.eslintrc.json

```diff
@@ -0,0 +1,42 @@
+{
+  "extends": "./node_modules/gts",
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  "plugins": ["node", "ban"],
+  "rules": {
+    "ban/ban": [
+      "error",
+      {
+        "name": "setTimeout",
+        "message": "WPT (Web Platform Tests) doesn't allow setTimeout."
+      }
+    ],
+    "linebreak-style": ["error", "unix"],
+    "no-console": "error",
+    "@typescript-eslint/typedef": [
+      "error",
+      {
+        "arrowParameter": false,
+        "memberVariableDeclaration": false
```

Comment:
This is to match what the current code does, but it overrides some of the ESLint recommended rules. They recommend being more strict with types everywhere, even things like:
class A {
  foo = 0; // needs type annotation
}

Reply:
Okay, will do in a follow-up